chemreg/lists/views.py

```diff
@@ -0,0 +1,9 @@
+from chemreg.jsonapi.views import ModelViewSet
+from chemreg.lists.models import ListType
+from chemreg.lists.serializers import ListTypeSerializer
+
+
+class ListTypeViewSet(ModelViewSet):
```

Comment:
```suggestion
from chemreg.common.mixins import DeprecateDeleteMixin

class ListTypeViewSet(DeprecateDeleteMixin, ModelViewSet):
```

Don't forget, we don't really want to delete these!